docs/how-to/create-initial-data-migrations.md

```diff
@@ -0,0 +1,111 @@
+- rows 2 to n are the initial data for the model we want to turn into a script.
+
+## Convert the data into JSON
+
+   1. Export the data from the Google [spreadsheet][pd-data-spreadsheet]
+      1. Find the sheet in the document containing the data to export. Let's use the `ProgramArea - Data` data as our example. It's
+      1. Make sure that the first row (column names) is frozen. Otherwise, freeze it by selecting the first row in the sheet, then Menu > View > Freeze > Up to row 1
+      1. Export to JSON. Menu > Export JSON > Export JSON for this sheet
+   1. Save the JSON into a file
+      1. Select and copy all the JSON text
+      1. Paste it into a new file and save it as [ModelNameInPascalCase]_export.json under app/core/initial_data/
+      1. The Pascal case is important in the next step to generate a python script to insert the data. It must match the model's class name for this to work.
+
+      **Potential data issue**
+      There was a problem with the JSON exporter where it omitted the underscore in `occ_code`. It should be fixed now but it's good to pay attention to other column name problems and fix them in the [Google Apps script][apps-script] in the [spreadsheet][pd-data-spreadsheet]. You will find out when the data insertion fails if there's a problem.
+## Convert JSON into Python script
+
+   1. Start Docker
+   1. From project root, run
+
+      ```bash
+      ./scripts/buildrun.sh
+      ```
+
+
+   1. Go to the project root and run this command
+
+      ```bash
+      docker-compose exec web python scripts/convert.py core/initial_data/ProgramArea_export.json
+      ```
+
+   1. Check that there's a new file called `app/core/scripts/programarea_seed.py` and that it looks correct
+      1. You can run it to verify, but will need to remove that data if you care about restoring the database state
+         1. Run this command to run the script
+
+            ```bash
+            docker-compose exec web python manage.py runscript programarea_seed
+            ```
+
+         1. To remove the data, go into the database and delete all rows from `core_programarea`
+
+            ```bash
+            docker-compose exec web python manage.py dbshell
+
+            # now we have a shell to the db
+
+            # see if all the seed data got inserted
+            select count(*) from core_programarea;
+            # shows 22 rows
+
+            delete from core_programarea;
+            # DELETE 22
+
+            select count(*) from core_programarea;
+            # shows 0 rows
+
+            # ctrl-d to exit dbshell
+            ```
+
+[pd-data-spreadsheet]: https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/
+[apps-script]: https://thenewstack.io/how-to-convert-google-spreadsheet-to-json-formatted-text/#:~:text=To%20do%20this,%20click%20Extensions,save%20your%20work%20so%20far.
+
+## Call Script From Migration
```

Comment:
Wasn't able to get this working. 

Ran `python manage.py migrate` inside the docker container but it did not populate any information into the db

Reply:
It might have already been added to the db. Maybe try `python manage.py migrate data 0001` to see if it rolls back the data. I'm not sure how to roll back past that.